i18n/events/open-spaces.i18n.js

```diff
@@ -4,81 +4,87 @@ export default genI18nMessages({
     'en-us': {
         title: 'Open Space',
         intro:
-            'Open Spaces are self-organizing meetup events that happen simultaneously with the main conference. ' +
-            'Open Spaces are organized during the period of PyCon by attendees, ' +
-            'which provides a way for you to define, organize, ' +
+            'Open Spaces are self-organizing meetup events that occur simultaneously with the main conference. ' +
```

Comment:
-> "Open Space is a self-organizing meetup event that occurs simultaneously with the main conference."